ui/src/components/textInputDialog.tsx

```diff
@@ -20,7 +21,9 @@ export function useTextInputDialog(
   const [open, setOpen] = useState(false);
   const show = () => setOpen(true);
 
-  const [text, setText] = useState(props.title);
+  const [text, setText] = useState(
+    (props.initialText = props.initialText || "")
```

Comment:
Why are you assigning to `props.initialText`? I don't see why it's necessary but even if it was it's confusing to do it inside of an expression.

Reply:
OMG!!! My fault, so stupid am I...